core/src/lib/tls_openssl.cc

```diff
@@ -317,7 +317,7 @@ void TlsOpenSsl::TlsBsockShutdown(BareosSocket* bsock)
 
   if (jcr && jcr->is_passive_client_connection_probing) { return; }
 
-  std::string message{_("TLS shutdown failure.")};
+  std::string message{T_("TLS shutdown failure.")};
```

Comment:
```suggestion
  const char* message = T_("TLS shutdown failure.");
```
This does not need to be changed, but feel free to do so.